cms/core/blocks/panels.py

```diff
@@ -37,3 +42,54 @@ class Meta:
         template = "templates/components/streamfield/information_panel.html"
         icon = "info-circle"
         label = _("Information Panel")
+
+
+class PreviousVersionBlock(blocks.IntegerBlock):
+    pass
+
+
+class CorrectionOrNoticeBlock(blocks.StructBlock):
+    when = blocks.DateTimeBlock()
+    text = blocks.RichTextBlock(features=settings.RICH_TEXT_BASIC)
+
+    class Meta:
+        abstract = True
+
+
+class CorrectionBlock(CorrectionOrNoticeBlock):
+    previous_version = PreviousVersionBlock(required=False)
+    frozen = blocks.BooleanBlock(required=False)
```

Comment:
Question: Should this have an explicit default of `false`?

Reply:
Added default value.